velox/docs/configs.rst

```diff
@@ -414,6 +414,16 @@ Each query can override the config by setting corresponding query session proper
      - bool
      - true
      - If true, the partition directory will be converted to lowercase when executing a table write operation.
+   * - allow-null-partition-keys
+     - allow_null_partition_keys
+     - bool
+     - true
+     - Determines whether null values for partition keys are allowed or not. if false, it will threw error message
```

Comment:
> if false, it will...

If not, fails with "Partition key must not be null" error message when writing data with null partition key.

Reply:
good advice, fixed.